TypeCobol/Tools/Options-Config/TypeCobolConfiguration.cs

```diff
@@ -96,7 +96,6 @@ public void LoadCpyCopyNameMap(string cpyCopyNameMapFilePath)
             }
         }
 #endif
-
```

Comment:
This can be reverted, to avoid noise change.

Reply:
Integrated in [7f2afb7](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/7f2afb778747d8f26ff0ce751781c2ab8845aa78)